scrapy/extensions/feedexport.py

```diff
@@ -57,7 +58,9 @@ def _store_in_thread(self, file):
 @implementer(IFeedStorage)
 class StdoutFeedStorage(object):
 
-    def __init__(self, uri, _stdout=sys.stdout):
+    def __init__(self, uri, _stdout=None):
+        if not _stdout:
+            _stdout = sys.stdout if six.PY2 else sys.stdout.buffer
```

Comment:
Btw, this is also used here https://github.com/scrapy/scrapy/blob/da83f2dca70020b991bafd296f6c7b9161e418b4/scrapy/commands/fetch.py#L45 - might be worth putting into ``scrapy.utils.python``?